cabot/cabotapp/views.py

```diff
@@ -481,6 +481,9 @@ def render_to_response(self, context, *args, **kwargs):
             context = {}
         context['checkresults'] = self.object.statuscheckresult_set.order_by(
             '-time_complete')[:100]
+        statuscheck_id = self.objects.statuscheckresult_set.first().status_check.id
```

Comment:
```suggestion
        statuscheck_id = self.object.id
```

Reply:
Ahh I have direct access to the check model in that view.  Cool!